src/app/editor/components/node-tags-bar/node-tags-bar.component.ts

```diff
@@ -47,7 +45,8 @@ export class NodeTagsBarComponent implements OnChanges {
         }
     }
 
-    onFilterChange(term: string) {
+    onFilterChange(term: string): void {
+        this.filterTerm = term;
 
         this.filteredTags = this.filterTags(term);
 
```

Comment:
Refactor this to get the `tags` array as seldom as possible - currently being done 2x per input event. Store as a private class field.